src/Core/Content/Product/SalesChannel/Listing/Processor/SortingListingProcessor.php

```diff
@@ -106,35 +105,18 @@ private function getAvailableSortings(Request $request, Context $context): Produ
 
             $filter = new EqualsAnyFilter('id', $availableSortingsById);
 
-            $availableSortingsByName = array_filter($availableSortingsFilter, fn ($filter) => !Uuid::isValid($filter));
```

Comment:
should this really be removed? :thinking:

Reply:
Yes the variable is no longer used 🤔

Reply:
but only because you remove the complete condition below. shouldn't only the feature condition removed there? 🤔

unfortunately this is exactly what @Phil23 recently wrote in slack 🙈

maybe we can find the person, who knows something about that? 😁

Reply:
the whole ifs are always false because !Feature::isActive('v6.7.0.0') is always false.

But to ensure it's correct, @keulinho was the one who added it please double check :)

Reply:
oh you are right :see_no_evil: 

I also checked the ticket and the issue. it should be fine like this :+1: